core/tests/test_utils.py

```diff
@@ -194,14 +236,24 @@ def _restore_stashed_user_env(self):
         for key in self.stashed_user_env:
             os.environ[key] = self.stashed_user_env[key]
 
-        self.stashed_user_env = None  # pylint: disable=attribute-defined-outside-init
+        self.stashed_user_env = None  
+        # pylint: disable=attribute-defined-outside-init
 
     def login(self, email, is_super_admin=False):
+        """Assign environment variables for login credentials of user.
+
+        Args:
+            email: str. The email address of the user.
+            is_super_admin: bool. Whether the user is a super admin.
+        """
         os.environ['USER_EMAIL'] = email
         os.environ['USER_ID'] = self.get_user_id_from_email(email)
         os.environ['USER_IS_ADMIN'] = '1' if is_super_admin else '0'
 
     def logout(self):
+        """Reassign environment variables of login credentials to null string
```

Comment:
Don't go too deep into implementation details in a docstring. I think something like `Clears the environment variables related to the user's login credentials.` should suffice.